lib/rules/no-pointer-down-event-binding.js

```diff
@@ -19,25 +20,27 @@ function isOnModifier(node) {
  * @param {object} node
  * @return {boolean}
  */
-function isActionModifer(node) {
+function isActionModifier(node) {
   return match(node, { path: { original: 'action' } });
 }
 
 /**
- * Check if an event name is a "down" event
+ * Check if an event name is a pointer "down" event
  *
  * @param {string} eventName
  * @return {boolean}
  */
-function isDownEvent(eventName) {
-  return eventName.toLowerCase().endsWith('down');
+function isPointerDownEvent(eventName) {
+  return ['mousedown', 'onmousedown', 'pointerdown', 'onpointerdown'].includes(
```

Comment:
Can you declare this array in a `const` outside the function so it doesn't get recreated each time the function is called?

Reply:
Yes, I have updated the rule to declare the pointer-down event bindings in a set called during rule setup.